components/technician/TechnicianVisitListItem.tsx

```diff
@@ -0,0 +1,17 @@
+import { enFullDate } from "@/lib/formatter"
+import { TechnicianVisit } from "@/types/content"
+import { Node } from "@/types/highLevel"
+import Link from "next/link"
+
+type Props = {
+  visit: Pick<Node<TechnicianVisit>, "created" | "id" | "title">
+}
+
+export default function TechnicianVisitListItem({ visit }: Props) {
+  return <Link key={visit.id} href={`/visit/${visit.id}`}>
+    <div className="px-6 py-2 bg-base-200 my-1">
+      <p className="font-semibold">{visit.title}</p>
+      <p className="text-sm">{enFullDate.format(new Date(visit.created))}</p>
```

Comment:
We should a utility function in `@/lib/formatter` so we could do something like:

```suggestion
      <p className="text-sm">{formatDate(visit.created, FULL_DATE)}</p>
```

Then handle whether it's English or Portuguese in that utility function (rather than forcing the developer to manually switch between `enFullDate` and `ptFullDate` depending on the current language. 

(Don't know if you already handled this in #4  )

Reply:
It'll be fixed in #5